lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -36,9 +36,45 @@ function collectImageElementInfo() {
       clientHeight: element.clientHeight,
       naturalWidth: element.naturalWidth,
       naturalHeight: element.naturalHeight,
+      isCss: false,
       isPicture: element.parentElement.tagName === 'PICTURE',
     };
   });
+
+  // Only match basic background-image: url("http://host/image.jpeg") declarations
+  const ALLOWED_IMAGE = /^\s*url\("([^"]+)"\)\s*$/;
```

Comment:
quotes aren't required inside of a url().. And i personally rarely use them.
also some people use single quotes.

so let's allow these combos.  this guy (thanks SO) seems good:

```regex
/(?:\(['"]?)(.*?)(?:['"]?\))/
```